e2e/claims.service.e2e.ts

```diff
@@ -113,7 +114,7 @@ jest.mock('../src/modules/messaging/messaging.service', () => {
   };
 });
 
-describe('Enrollment claim tests', () => {
+describe.skip('Enrollment claim tests', () => {
```

Comment:
Just a note to remove the `skip` before merging
```suggestion
describe('Enrolment claim tests', () => {
```